app/sidecar/base.toml

```diff
@@ -1114,6 +1119,7 @@ owner = {name = "control_plane_agent", notification = "socket"}
 port = 11111
 tx = { packets = 3, bytes = 2048 }
 rx = { packets = 3, bytes = 2048 }
+accept-untrusted = true
```

Comment:
So, how would you feel about `allow-untrusted`? I bring this up because
- this setting applies to both transmit and receive, whereas normally in a firewall config I think of "accept" as applying to inbound traffic only
- "allow" puts it more in line with allow/deny terminology in packet filters

Reply:
I like it, done in 677894e28891